src/controllers/user.js

```diff
@@ -32,15 +32,19 @@ export const create = async (req, res) => {
 
 export const getById = async (req, res) => {
   const id = parseInt(req.params.id)
+  const { role } = req.user
 
   try {
-    const foundUser = await User.findById(id)
-
+    let foundUser = {}
+    if (role === 'TEACHER') {
```

Comment:
Create a `isTeacher = role === 'TEACHER'` variable that is checked in the if statement instead, reads better